cmd/gofr/entity/entity.go

```diff
@@ -213,7 +215,7 @@ func addConsumer(f fileSystem, projectDirectory, entity string) error {
 		return err
 	}
 
-	filePtr, err := f.OpenFile(entity+".go", os.O_APPEND|os.O_CREATE|os.O_WRONLY, migration.RWMode)
+	filePtr, err := f.OpenFile(entity+goExtension, os.O_APPEND|os.O_CREATE|os.O_WRONLY, migration.RWMode)
```

Comment:
Shouldn't we rename it to extensionGo, such that we have more such in future we can follow the same way, and IDE would give suggestions

Reply:
Sure will rename that